packages/flutter/lib/src/widgets/page_storage.dart

```diff
@@ -130,6 +130,18 @@ class PageStorageBucket {
 }
 
 /// A widget that establishes a page storage bucket for this widget subtree.
+/// 
+/// ```dart
```

Comment:
This sample should follow the sample style we use everywhere in the repository: It needs to be wrapped in `{@tool sample ...}` and have an explanation what the example actually shows.

Reply:
I have corrected it